examples/ibm-cis/README.md

```diff
@@ -21,6 +21,7 @@ These types of resources are supported:
 * [ CIS Custom Page](https://cloud.ibm.com/docs/terraform?topic=terraform-cis-resources#cis-custom-page)
 * [ CIS Page Rule](https://cloud.ibm.com/docs/terraform?topic=terraform-cis-resources#cis-page-rule)
 * [ CIS WAF Package ](https://cloud.ibm.com/docs/terraform?topic=terraform-cis-resources#cis-waf-package)
+* [ CIS Certificate Order](https://cloud.ibm.com/docs/terraform?topic=terraform-cis-resources#cis-certificate-order)
```

Comment:
Do we have one more resource for uploading in development?

Reply:
yes. It is under development